### PR TITLE
Chore: Update README.md - Cleanup

### PR DESCRIPTION
**Pull request by iAmNathanJ**
@catheraaine @BC3209 @boomingbrake
README had some spelling and formatting errors on my part. I think this cleans it up. Let me know if anyone sees anything else. 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,21 +27,22 @@ Using Rails and PostgreSQL
 
 ### Heroku Setup
 This setup assumes you have a version of the app in a GitHub repository.
-1. [crete a new app from the dashboard at Heroku](https://dashboard.heroku.com/new)
-2. Under the the deploy section of the newly created app dashboard do:
+
+1. [Create a new app from the Heroku dashboard](https://dashboard.heroku.com/new)
+2. Under the the deploy section of the newly created app dashboard:
```
Comment (boomingbrake) [inline README.md]: the x2

Comment (iAmNathanJ) [inline README.md]: @boomingbrake I don't understand. What am I missing?

**Comment by iAmNathanJ** [comment]
@boomingbrake That should be it, I think... *I think*.

**Comment by boomingbrake** [comment]
Looks good to me!  Straight forward and understandable, good write up!

**Pull request merged**